### ISSUE TITLE
Commando broken in 10.2 build 89795

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#2292]
### Short Summary

Commando never populates when I open it in the latest 10.2 nightly.

### Additional Information

```
[2017-01-30 10:01:22,372] [ERROR] console-logger: TypeError: resultElem.childCount is not a function (2) in chrome://commando/content/sdk/commando.js:1194
Traceback (most recent call last):
  File "chrome://commando/content/sdk/commando.js", line 1194, in 
```

**Comment by Naatan** [comment]
Fixed with 10beba3bbddfca76274e8a089a8266384f9af9dd

**Comment by hoppfrosch** [comment]
Fix confirmed 
Couldn't find this bug because "Commando" is a very generic search term. I'd prefer  "Go To Anything" over "Commando", as its more precise and is easier to search for (for example googling for "KomodoIde Commando" leads to a much more blurred result  as googling for "Komodide Goto anything". And the thing is not a "commando" but a commando central (also the thing is not a "Go to Anything") ;-)
